### ISSUE TITLE
Sharpshooter is missing a dependency on Kali image

### ISSUE THREAD

**Issue by justinforbes** [cisagov/cool-system#121]
# 🐛 Bug Report

Sharpshooter is missing the python2 package jsmin on the Kali image

## To Reproduce

Attempting to generate a payload with Sharpshooter give an error:
python2: ImportError: No module named jsmin

## Expected behavior

Please add the jsmin python2 package to the base Kali image

**Comment by jsf9k** [comment]
There is no Kali package that satisfies this:
```console
$ sudo apt-cache search jsmin
libghc-hjsmin-dev - JavaScript minifier
libghc-hjsmin-doc - JavaScript minifier; documentation
libghc-hjsmin-prof - JavaScript minifier; profiling libraries
python3-jsmin - JavaScript minifier written in Python - Python 3.x
python3-rjsmin - javascript minifier written in Python - Python 3.x
yui-compressor - JavaScript/CSS minifier
```

[mdsecactivebreach/SharpShooter](https://github.com/mdsecactivebreach/SharpShooter) does contain a `requirements.txt` file, though.

**Comment by jsf9k** [comment]
See also the discussion in [cisagov/cool-systems#120](https://github.com/cisagov/cool-systems#120).

**Comment by jsf9k** [comment]
Will be resolved by #120/[cisagov/ansible-role-kali#12](https://github.com/cisagov/ansible-role-kali/issues/12).
